### DISCUSSION TITLE
Unnecessary recommendation filters ?

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [cloud-carbon-footprint/cloud-carbon-footprint#1338] [Ideas]
AWS offers recommendation findings to change the instance type, especially to graviton. The finding type is 'OPTIMIZED' in that case. CCF filters for finding type 'OVER_PROVISIONED', 'NOTOPTIMIZED' [here](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/trunk/packages/aws/src/lib/Recommendations/ComputeOptimizerRecommendations.ts):
```
const includedRecommendationTypes = ['OVER_PROVISIONED', 'NOTOPTIMIZED']
...
if (
              includedRecommendationTypes.includes(
                recommendationType.toUpperCase(),
              )
            ) ...
```
Is that filtering really necessary? OPTIMIZED recos seem valid and good.

**Comment by ccasher** [comment]
Hi @jochen-schuettler, thanks for posting! At our time of implementation, our thought process was that the "OPTIMIZED" recommendations may not always correlate to carbon emissions savings (and sometimes actually raise them). If we find this would be a valuable inclusion to have for ARM based processors, then we could definitely consider it.
